skimage/exposure/exposure.py

```diff
@@ -463,3 +463,28 @@ def adjust_sigmoid(image, cutoff=0.5, gain=10, inv=False):
 
     out = (1 / (1 + np.exp(gain * (cutoff - image / scale)))) * scale
     return dtype(out)
+
+
+def is_low_contrast(image, threshold_percent=0.5, lower_bound=1,
+                    upper_bound=99, method='linear'):
+    """Detemine if an image is low contrast.
+
+    Parameters
+    ----------
+    image : array-like
+        The image under test.
+    threshold_percent : float, optional
+        The low contrast threshold in percent.
+    lower_bound : float, optional
+        The lower bound percentile of the image to use.
+    upper_bound : float, optional
+        The upper bound percentile of the image to use.
+    method : str, optional
+        The contrast determination method.  Right now the only available
+        option is "linear".
+    """
```

Comment:
Also, a couple of simple doctests (using "images" generated with `np.arange` or `np.linspace`) might be really good here to illustrate the parameters. e.g. you can have an image that's np.linspace(0, 0.04, 100), then set image[-1] = 1, and see how the result of `is_low_contrast` changes when changing `upper_percentile` from 99 to 100.